nodes/pods/nodes-pods-autoscaling-custom.adoc

```diff
@@ -25,16 +25,17 @@ include::snippets/technology-preview.adoc[leveloffset=+0]
 // modules required to cover the user story. You can also include other
 // assemblies.
 
-include::modules/nodes-pods-autoscaling-custom-rn.adoc[leveloffset=+1]
+include::modules/nodes-pods-autoscaling-custom-rn-210.adoc[leveloffset=+1]
 
 .Additional resources
 
-* xref:../../operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Changing the update channel for an Operator]
-* xref:../../nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-gather_nodes-pods-autoscaling-custom[Gathering debugging data]
-* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-audit_nodes-pods-autoscaling-custom[Configuring audit logging]
-* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-trigger-cpu_nodes-pods-autoscaling-custom[Understanding the CPU trigger]
-* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-trigger-memory_nodes-pods-autoscaling-custom[Understanding the memory trigger]
-* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-trigger-kafka_nodes-pods-autoscaling-custom[Understanding the Kafka trigger]
+* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-metrics-access_nodes-pods-autoscaling-custom[Accessing performance metrics]
+* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-pausing_nodes-pods-autoscaling-custom[Pausing the custom metrics autoscaler for a workload]
+* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-creating-workload_nodes-pods-autoscaling-custom[Adding a custom metrics autoscaler to a workload]
+* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-about_nodes-pods-autoscaling-custom[Understanding the custom metrics autoscaler]
```

Comment:
The Accessing performance metrics link will not work until https://github.com/openshift/openshift-docs/pull/59040 merged